Assets/Scripts/InGame/Game.cs

```diff
@@ -130,22 +146,66 @@ private void StartPoker()
         if(!PhotonNetwork.IsMasterClient)
             return;
         
+        // create game account /v1/game/create-account here
+        AccountCreationRequest request = new AccountCreationRequest
+        {
+            game_player_ids = new ulong[] { 273475126759447508 }, // a sample player account id
+            big_blind = 100,  
+            small_blind = 50,  
+            buy_in = 200  
+        };
+        StartCoroutine(CreateAzeGameAccount(request));
+        
         GameEvents.NetworkGameplayEvents.OnUpdatePlayersView.Raise();
         
         StartNextStage(GameStage.PreFlop);
     }
 
+    private IEnumerator CreateAzeGameAccount(AccountCreationRequest request)
+    {
+        Debug.Log("Creating game account...");
+        string url = "http://127.0.0.1:8000/v1/game/create-account";
```

Comment:
I think it would be good to have some sort of configuration file for the base url and routes.

Reply:
Please explain a bit more.

Reply:
For example in aze-server we have `miden-client.toml` which contains the configuration for store and we generally have `config.json` files for js project configs.